packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -93,7 +93,11 @@ export class OfflineQueueLink extends ApolloLink {
   }
 
   private enqueue(entry: OperationQueueEntry) {
-    this.opQueue = squashOperations(entry, this.opQueue);
+    if (hasDirectives([Directives.NO_SQUASH], entry.operation.query)) {
```

Comment:
Ignore me here, but if we adding feature for squashing and we have separate file then this change should probably land there. It's good pattern but let's not change it